facial-age-estimator/src/facial-age-estimator.js

```diff
@@ -0,0 +1,40 @@
+import { preprocess } from '../src/facial-age-estimator-input'
```

Comment:
since they are all in `src/` you don't necessarily need to go up one level and back into `src/` you can just do
```
import { preprocess } from './facial-age-estimator-input'
import { load, inference } from './facial-age-estimator-model'
import { postprocess } from './facial-age-estimator-output'
```